src/thermobench.cpp

```diff
@@ -48,6 +48,15 @@ struct sensor {
     const char *units;
 };
 
+#define MAX_CPUS 256
+struct cpu_usage {
+    unsigned n_cpus;
+    double usage[MAX_CPUS];
+    unsigned cpu_idx[MAX_CPUS];
+    unsigned curr_procstat[MAX_CPUS][8];
+    unsigned last_procstat[MAX_CPUS][8];
+} cpu_usage;
```

Comment:
Make this structure for a single CPU and define the variable as `cpu_usage[MAX_CPUS]`.